spec/parsers/bulkrax/bagit_parser_spec.rb

```diff
@@ -5,6 +5,34 @@
 
 module Bulkrax
   RSpec.describe BagitParser do
+    describe '#total' do
+      context 'while exporting' do
+        subject { described_class.new(exporter) }
+        let(:exporter) { FactoryBot.create(:bulkrax_exporter_worktype_bagit) }
+        let(:bulkrax_exporter_run) { FactoryBot.create(:bulkrax_exporter_run, exporter: exporter) }
+        let(:work_ids_solr) { [OpenStruct.new(id: SecureRandom.alphanumeric(9)), OpenStruct.new(id: SecureRandom.alphanumeric(9))] }
+        let(:collection_ids_solr) { [OpenStruct.new(id: SecureRandom.alphanumeric(9))] }
+        let(:file_set_ids_solr) { [OpenStruct.new(id: SecureRandom.alphanumeric(9)), OpenStruct.new(id: SecureRandom.alphanumeric(9)), OpenStruct.new(id: SecureRandom.alphanumeric(9))] }
```

Comment:
this logic is already in the `describe '#create_new_entries'...` section of this file. can we just move it out so that it's usable more than once, instead of duplicating it?